src/addons/MountNode/MountNode.js

```diff
@@ -0,0 +1,55 @@
+import PropTypes from 'prop-types'
+import React, {Component} from 'react'
+
+import {TYPES} from '../../lib/META'
+import handleChange from './lib/handleChange'
+import NodeRegistry from './lib/NodeRegistry'
+
+const nodeRegistry = new NodeRegistry()
+
+/**
+ * Component.
+ */
+export default class MountNode extends Component {
+  static propTypes = {
+    /** Primary content. */
+    className: PropTypes.string,
+
+    node: PropTypes.node,
+  }
+
+  static _meta = {
+    name: 'MountNode',
+    type: TYPES.ADDON,
+  }
+
+  shouldComponentUpdate({className: next}) {
+    const {className: current} = this.props
+
+    return next !== current
```

Comment:
I know we've started this pattern of renaming destructured keys elsewhere but I'd prefer more explicit names like `nextClassName` and `currClassName`.